README.md

```diff
@@ -116,3 +122,9 @@ Several scripts form a pipeline for updating the database. Here is the data flow
                         └──────────────────────┘ │
                           └──────────────────────┘
 ```
+
+> **NOTES**:
+>
+> * These scripts also garbage-collect unreferenced data.
+> * Instead of calling each of these scripts directly, you can call `./fetchAndCalculateAll.sh`
```

Comment:
To me it's clear from there https://github.com/ghuser-io/db/pull/1/commits/7c6efecdb57c350dfb85046be1f8f152bb2ec2e5#diff-04c6e90faac2675aa89e2176d2eec7d8R52

Reply:
True, although I wonder if someone else could understand "Refresh and clean data for all tracked users" like not refreshing the orgs and repos maybe?